doc/modules/linear_model.rst

```diff
@@ -1135,17 +1149,29 @@ in the following figure,
    mass at :math:`Y=0` for the Poisson distribution and the Tweedie (power=1.5)
    distribution, but not for the Gamma distribution which has a strictly
    positive target domain.
+   
+.. figure:: ./glm_data/bernoulli_distribution.png
```

Comment:
This figure should be improved. If `Y` is categorical, the x-axis should reflect that.
I'm not sure, if we need this figure at all.

Reply:
I added this figure for completeness. But I do see your point, this figure is not needed. A short note about the Bernoulli distribution instead would be a better idea.